org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizard.java

```diff
@@ -34,8 +35,16 @@
 	private DotnetNewWizardPage wizardPage;
 	@Override
 	public void init(IWorkbench workbench, IStructuredSelection selection) {
-		// TODO
 		wizardPage = new DotnetNewWizardPage();
+
+		Object firstElement = selection.getFirstElement();
+		if (firstElement instanceof IResource) {
```

Comment:
The link you sent me has logic for getting selected working sets, but I do not believe it had logic for getting the working set of a selected item. Do you know a way of getting working sets from a project better than going through working sets looking for the project?